beacon_node/network/src/sync/range_sync/chain.rs

```diff
@@ -550,9 +551,34 @@ impl<T: BeaconChainTypes> SyncingChain<T> {
             BatchProcessResult::FaultyFailure {
                 imported_blocks,
                 penalty,
+                faulty_component,
             } => {
+                let Some(batch_peers) = batch.processing_peers() else {
+                    crit!(
+                        current_state = ?batch.state(),
+                        "Inconsistent state, batch must have been in processing state"
+                    );
```

Comment:
We don't log a crit in other invalid state errors, is this log necessary?

Reply:
Changed to warn, something is inconsistent if we reach here so i'd like to keep the log